crates/re_viewer/src/viewer_analytics.rs

```diff
@@ -42,8 +41,10 @@ impl ViewerAnalytics {
         let mut analytics = Self { analytics };
 
         #[cfg(target_arch = "wasm32")]
-        if let Some(url) = analytics_url {
-            analytics.register("url", url);
+        if let Some(location) = startup_options.location.as_ref() {
+            if location.hostname.contains("rerun.io") {
```

Comment:
I think we should try to be water tight here

```suggestion
            if location.hostname == "rerun.io" || location.hostname.ends_with(".rerun.io") {
```